core/market/decentralized/src/config.rs

```diff
@@ -0,0 +1,48 @@
+use chrono::Duration;
+
+/// TODO: Decide where should this config be loaded from.
+///  We could deserialize it from .json file or use structopt and
+///  configure market through env variables.
+pub struct Config {
```

Comment:
@jiivan pls look at this. We (with @nieznanysprawiciel ) would like to introduce some kind of marriage of this approach and yours from #453